### DISCUSSION TITLE
表情显示不出

### DISCUSSION THREAD

**Discussion by Wu-Haitao** [walinejs/waline#272] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/59434361/118524849-2c753b80-b771-11eb-9588-927d4af07883.png)

之前用着没问题..今天突然发现表情显示不出来了，好奇怪哦，有什么解决方法吗？

**Comment by Wu-Haitao** [comment, marked as answer]
找到问题了，原来是忘记升级服务端了..
> 进入到对应的 GitHub 仓库中，修改 package.json 文件中的 @waline/vercel 版本号为最新版本即可。

**Reply by CloverWishes** [reply]
原来如此，我之前遇到同样的问题了，一直不知道是怎么回事
